xonsh/environ.py

```diff
@@ -926,6 +928,15 @@ def _current_job():
     return ''
 
 
+def env_name(pre_chars='(', post_chars=') '):
+    """Extract the current environment name from $VIRTUAL_ENV."""
+    env_path = builtins.__xonsh_env__.get('VIRTUAL_ENV', '')
```

Comment:
This is totally reasonable. Here you go: https://github.com/moigagoo/xonsh/commit/5bb1ea76a33e4f9c7f82849d487caf67623ebdd1